### ISSUE TITLE
Model Training + Evaluation - Skin Cancer MNIST HAM10000 (Week 6)

### ISSUE THREAD

**Issue by margoglvz** [CareTech-General/general-project-fall25#27]
The current issue is that CareTech would like to **build a model to predict melanoma from mole images.** 

Here is the dataset we would like you to use:
[HAM10000](https://www.kaggle.com/datasets/kmader/skin-cancer-mnist-ham10000)

Now that you have made your observations and identified or finished your data preprocessing, next step is **model training**! We will be using **Convolutional Neural Networks** which is what we went over during our previous General Meetings. Through testing, see what types of features or tweaks you can make to your CNN to increase your validation accuracy. 

Downloading the Dataset in Google CoLab
Run this command in a code block to download the dataset in the Google CoLab space:

`import kagglehub`

`# Download latest version`
`path = kagglehub.dataset_download("paultimothymooney/blood-cells")`
`print("Path to dataset files:", path)`
